back/app/serializers/web_api/v1/participation_context_serializer.rb

```diff
@@ -46,7 +46,7 @@ class WebApi::V1::ParticipationContextSerializer < WebApi::V1::BaseSerializer
     && context.voting? \
     && (
       (current_user(params) && UserRoleService.new.can_moderate?(context, current_user(params))) \
-      || TimelineService.new.phase_is_complete?(context)
+      || (context.phase? && TimelineService.new.phase_is_complete?(context))
```

Comment:
Wouldn't `context.participation_context?` on line 45 always return false for projects now? If that would be case, should we consider moving the votes_count to the phase serializer?

Reply:
Yes, this will be the last PR in the process where I will move methods/objects from the participation context to the phase where they are no longer applicable to both